src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -423,5 +423,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
             Return member
         End Function
+
+        <Extension>
+        Friend Function ContainsTupleNames(member As Symbol) As Boolean
+            Select Case member.Kind
+                Case SymbolKind.Method
+                    Dim method = DirectCast(member, MethodSymbol)
+                    Return method.ReturnType.ContainsTupleNames() OrElse method.Parameters.Any(Function(p) p.Type.ContainsTupleNames())
+                Case SymbolKind.Property
+                    Return DirectCast(member, PropertySymbol).Type.ContainsTupleNames()
```

Comment:
Need to check property parameters as well. Consider extracting a method to check `ImmutableArray(Of ParameterSymbol)`.